### DISCUSSION TITLE
Assets always public

### DISCUSSION THREAD

**Discussion by matheus-rossi** [requarks/wiki#6839] [Help / Questions] [answered]
Just deployed WikiJs and configured login ONLY via google.
The login is working perfectly, only people from my organization can access it.

Just one thing i noticed, even if the content is blocked by login, are all assets public?

I can access the images without login, for example:

http://my-instance.us-east-1.elb.amazonaws.com/sap_hana_dbeaver.png

Is it correct ? How can i protect my assets from being public avaliable ?

**Comment by mittkkoo** [comment]
I can't reproduce this on my side. Maybe you have to check the settings of your webserver.

**Reply by matheus-rossi** [reply]
we deployed the app with the helm chart
Any tag i need to change ?

**Comment by griznog** [comment, marked as answer]
To block public access I had to go into the Guest group and remove access to all read permissions. Check your Guest group to see what access it has.

**Reply by matheus-rossi** [reply]
Worked like a charm, thanks
